lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -92,6 +92,12 @@
           - Do not use cache when building an image.
         type: bool
         default: no
+      extra_hosts:
+        description:
+          - Extra hosts to add to /etc/hosts in building containers, as a mapping of hostname to IP address.
+        type: dict
+        default: {}
+        version_added: "2.8.3"
```

Comment:
`version_added` needs to be `2.9`. We only add features in major versions.

Reply:
Additionally the `argument_spec` needs to include `extra_hosts` which it does not currently.

Reply:
Also `option_minimal_versions` needs to be extended, since this option potentially needs a new enough docker-py and/or docker daemon version.